Makefile

```diff
@@ -54,10 +61,11 @@ OBJS = \
 	elf.o \
 	main.o \
 	syscall.o \
-	syscall_sdl.o
+	syscall_sdl.o 
```

Comment:
Don't change this line. You should minimize the necessary changes.

Reply:
I think this is my mistake that type 'space' in this line. I will avoid these mistakes next time.